knime_extension/src/nodes/spatialnetwork.py

```diff
@@ -289,6 +345,7 @@ def fetch_google_od(download_link):
             nt_duration = 0
             nt_distance = 0
             try:
+                # TODO: Add advanced timeout parameter
```

Comment:
See todo

Reply:
Please also change the library to the more common request library

Reply:
library changed